cmd/update/volume_move.go

```diff
@@ -108,14 +105,43 @@ func (c *volumeMoveCommand) runWithCtx(ctx context.Context, cmd *cobra.Command,
 	}
 
 	if !isMovingMaster && hasReplicaOnTargetNode {
-		c.display.TextMessage(ctx, c.writer, "volume already has a replica on both [source] and [destination] nodes, nothing to do")
+		c.display.TextMessage(ctx, c.writer, "Volume already has a replica on both [source] and [destination] nodes, nothing to do")
 		return nil
 	}
 
+	// after we know a valid scenario has been triggered we return what will
+	// happen back to the user in a user-friendly msg
+	if isMovingMaster {
+		if hasReplicaOnTargetNode {
+			c.display.TextMessage(ctx, c.writer, "Detected: promoting replica")
+		} else {
+			c.display.TextMessage(ctx, c.writer, "Detected: moving master")
+		}
+	} else {
+		c.display.TextMessage(ctx, c.writer, "Detected: moving replica")
+	}
```

Comment:
`Detected` might be a bit confusing for the user.`Performing action` could be better